22.md

```diff
@@ -0,0 +1,192 @@
+NIP-22
+======
+
+Comment
+-------
+
+`draft` `optional`
+
+A comment is a threading note always scoped to a root event or an `S`-tag.
+
+It uses `kind:1111` with plaintext `.content` (no HTML, Markdown, or other formatting).
+
+Comments MUST point to the root scope using uppercase tag names (e.g. `K`, `E`, `A` or `S`)
+and MUST point to the parent item with lowercase ones (e.g. `k`, `e`, `a` or `s`).
+
+```js
+{
+  kind: 1111,
+  content: '<comment>',
+  tags: [
+    // root scope: event addresses, event ids, or S-tags.
+    ["<A, E, S>", "<address, id or S-value>", "<relay or web page hint>", "<root event's pubkey, if an E tag>"],
+    // the root kind or type if r.
+    ["K", "<root kind>"],
+
+    // parent item: event addresses, event ids, or s-tags.
+    ["<a, e, s>", "<address, id or s-value>", "<relay or web page hint>", "<parent event's pubkey, if an e tag>"],
+    // parent item kind
+    ["k", "<parent comment kind>"]
+  ]
+  // other fields
+}
+```
+
+Tags `K` and `k` MUST be present to define the event kind of the root and the parent items.
+
+`S` and `s` tags create scopes for hashtags, geohashes, URLs, and other external identifiers. They accept the following values:
+
+| Type | `S`/`s` tag | `K`/`k` tag |
+|- | - | - |
+| URLs | "r:`<URL, normalized, no fragment>`" | "r:`<scheme-host, normalized>`" |
+| Hashtags | "t:`<topic, lowercase>`" | "#:t" |
+| Geohashes| "g:`<geohash, lowercase>`" | "#:g" |
+| Podcast Feeds | "i:podcast:guid:`<id, lowercase>`" | "i:podcast:guid" |
+| Podcast Episodes | "i:podcast:item:guid:`<id, lowercase>`" | "i:podcast:item:guid" |
+| Books | "i:isbn:`<id, hyphenated>`" | "i:isbn" |
+| Movies | "i:isan:`<id, lowercase>`" | "i:isan" |
+| Papers | "i:doi:`<id, lowercase>`" | "i:doi" |
+
+
+`c` tags MAY be used when citing events in the `.content`.
+
+```json
+["c", "<event-id> or <event-address>", "<relay-url>", "<pubkey-if-a-regular-event>"]
+```
+
+`p` tags SHOULD be used when mentioning pubkeys in the `.content`.
+
+```json
+["p", "<pubkey>"]
```

Comment:
I don't think you need to clarify this. There is no difference to any other nostr event.

Reply:
So, you want only people that were directly mentioned? I don't think that restriction makes sense. People should be able to use `p` freely (to notify everybody needed based on the root scope, for instance)